appveyor.yml

```diff
@@ -0,0 +1,33 @@
+# https://www.appveyor.com/docs/appveyor-yml
+
+version: "{build}"
+
+clone_depth: 10
+
+environment:
+  matrix:
+    - nodejs_version: "10"
+    - nodejs_version: "8"
+    - nodejs_version: "6"
+
+install:
+  - ps: Install-Product node $env:nodejs_version
+  - yarn install --ignore-engines
+
+build: off
+
+test_script:
+  - set PATH=C:\msys64\usr\bin;%PATH%
+  - node -v
+  - yarn -v
+  - make lint
```

Comment:
I would remove linting, the platform has no impact

Reply:
ok